scss/vendor/_rfs.scss

```diff
@@ -9,6 +9,7 @@
 // Configuration
```

Comment:
RFS is a vendor, please do not edit it directly. It needs to be updated from upstream, see [RFS repository](https://github.com/twbs/rfs).

Reply:
But it doesn't say inside which exact version of rfs we have here? Should I update the latest? If so, it will pull in a lot more changes to it...this way, it would just fix the warnings...and leave functionality/behavior the same

Reply:
Leave RFS for now, we'll upgrade it in another PR. @XhmikosR Does this sound right to you?